src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/tests/test_api_check.py

```diff
@@ -29,6 +29,11 @@ def setUp(self):
     def tearDown(self):
         pass
 
+
+    def test_parse_resource(self):
+        parts = parse_resource_id('/subscriptions/00000/resourcegroups/bocconitestlabrg138089/providers/microsoft.devtestlab/labs/bocconitestlab/virtualmachines/tasktest1')
```

Comment:
Do we also need test with `G` in `resourceGroup` or it's already covered in other test?